tools/run_cargo_generate-lockfile.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Runs `cargo generate-lockfile` (which generates and/or updates the Cargo.lock
```

Comment:
hrm, it would be better to introduce [Cargo workspace](https://doc.rust-lang.org/cargo/reference/manifest.html#the-workspace-sectionl) to share `Cargo.lock` in `tock`, however I guess the boards wouldn't compile without the additional tricks that the `Makesfiles` is doing but it would be possible to leave those out.

Reply:
Yeah, we've talked about workspaces off and on a bunch, but the tooling just isn't quite flexible enough yet to be compatible with Tock's layout.